File: sdks/ruby/lib/cyberarmor/providers/microsoft.rb
# frozen_string_literal: true

module CyberArmor
  module Providers
    class Microsoft < OpenAICompatible
      def initialize(client:, openai:, tenant_id:, model: 'phi-4')
        super(client: client, openai: openai, tenant_id: tenant_id, provider: 'microsoft', model: model)
      end
    end
  end
end

File: sdks/php/src/Providers/Microsoft.php
<?php

declare(strict_types=1);

namespace CyberArmor\Providers;

use CyberArmor\Client;

final class Microsoft extends OpenAICompatible
{
    public function __construct(Client $client, object $openai, string $tenantId, string $model = 'phi-4')
    {
        parent::__construct($client, $openai, $tenantId, 'microsoft', $model);
    }
}

